ephios/templates/base.html

```diff
@@ -102,6 +102,14 @@
             {% block content %}{% endblock %}
         </div>
     </div>
+    <footer class="footer text-center mt-auto py-3">
```

Comment:
i'd prefer to have a footer that sticks to the bottom of the viewport (like so: https://getbootstrap.com/docs/4.1/examples/sticky-footer/)

Reply:
(but with your design, their big grey bar is ugly)

Reply:
I'll try :)